lib/workers/pr/index.ts

```diff
@@ -140,26 +126,19 @@ export async function ensurePr(
         config.forcePr = true;
       }
     }
-    if (
-      config.forcePr ||
-      [
-        BRANCH_STATUS_FAILURE,
-        BRANCH_STATUS_ERROR,
-        BRANCH_STATUS_FAILED,
-      ].includes(await getBranchStatus())
-    ) {
+    if (config.forcePr || (await getBranchStatus()) === BranchStatus.red) {
       logger.debug(`Branch tests failed, so will create PR`);
     } else {
       return null;
     }
   }
   if (config.prCreation === 'status-success') {
     logger.debug('Checking branch combined status');
-    if ((await getBranchStatus()) !== BRANCH_STATUS_SUCCESS) {
+    if ((await getBranchStatus()) !== BranchStatus.green) {
       logger.debug(
         `Branch status is "${await getBranchStatus()}" - not creating PR`
       );
-      return BRANCH_STATUS_PENDING;
+      return 'pending';
```

Comment:
I think we can add another enum for those pr states

Reply:
Yes I was thinking `BranchResult`

Reply:
Should be done in separate pr. 🙃